package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "Magnet",
+  "name": "project-magnet",
```

Comment:
Yeh, I think we've already got a mismatch because we're using org.mozilla namespace.  But I'm not going to touch this as there is all sorts of weird stuff overwritten and changed when running `react-native upgrade`